docs/domains/Domain.json

```diff
@@ -223,18 +252,23 @@
           "description": "Experimental feature configurations.",
           "$ref": "#/definitions/Experimental"
         },
-        "opssWalletSecret": {
-          "description": "opss key passphrase.",
-          "$ref": "https://github.com/garethr/kubernetes-json-schema/blob/master/v1.13.5/_definitions.json#/definitions/io.k8s.api.core.v1.SecretReference"
-        },
         "serverService": {
           "description": "Customization affecting ClusterIP Kubernetes services for WebLogic Server instances.",
           "$ref": "#/definitions/ServerService"
         },
         "domainHome": {
-          "description": "The folder for the WebLogic Domain. Not required. Defaults to /shared/domains/domains/domainUID if domainHomeInImage is false. Defaults to /u01/oracle/user_projects/domains/ if domainHomeInImage is true.",
+          "description": "The folder for the WebLogic Domain. Not required. Defaults to /shared/domains/domains/domainUID if domainHomeSourceType is not Image. Defaults to /u01/oracle/user_projects/domains/ if domainHomeSourceType is Image.",
```

Comment:
Defaults to /u01/oracle/user_projects/domains/ if domainHomeSourceType is Image.", --> Defaults to /u01/oracle/user_projects/domains/ if domainHomeSourceType is Image or FromModel.",

Reply:
Pretty sure - we should verify with Johnny.

Reply:
Change:
Default for PV remains same as before.
Default for Image remains same as before.
Default for From Model is:  /u01/domains/DOMAIN_NAME